doc/appendix-de-kassensichv/reference-tables/service-status-ftstate.md

```diff
@@ -11,9 +11,5 @@ The table below describes its supported statuses for the ftState field following
 
 | **Value**            | **Description**                                                                                     | **Middleware Version** |
 |----------------------|-----------------------------------------------------------------------------------------------------|---------------------|
-| `0x4445000000000002` | The security mechanism was not able to communicate with the TSE device for at least one cycle. If this is the case, no more communication attempts are done to avoid long waiting times for each Receipt request/Receipt response sequence. To leave this state, a Zero-Receipt must be sent, which forces a communication retry towards the TSE device.<br /> Receipts created in a state where no communication is possible with the TSE device are protected by the security mechanism of fiskaltrust. \| 1.0 | 1.0                 |
-| `0xXXXX000000000004` | Reserved                                                                                            |                     |
-| `0xXXXX000000000010` | Reserved                                                                                            |                     |
-| `0xXXXX000000000020` | Reserved                                                                                            |                     |
-| `0xXXXX000000000080` | Reserved                                                                                            |                     |
-| `0xXXXX000000000100` | The Middleware is in the process of switching SCUs. This state is returned in case any receipts are processed between the _initialize-switch receipt_ and the _finish-switch receipt_. These receipts are protected by the fiskaltrust.SecurityMechanism, but not sent to any TSE, as no SCU is connected at this point. |   1.3.19                  |
+| `0x4445000000000002` | The security mechanism was not able to communicate with the TSE device for at least one cycle. If this is the case, no more communication attempts are done to avoid long waiting times for each Receipt request/Receipt response sequence. To leave this state, a Zero-Receipt must be sent, which forces a communication retry towards the TSE device.<br /> Receipts created in a state where no communication is possible with the TSE device are protected by the security mechanism of fiskaltrust. \ | 1.0                 |
```

Comment:
is the "\"  in the end intentionally or should we remove it?

Reply:
Sorry, it is a typo. It has been corrected with the commit: bc0cfa6774d1aeee984e67ee29f6989f3bc22650